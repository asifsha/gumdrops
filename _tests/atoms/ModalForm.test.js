/* globals mount */
import React from 'react';
import ModalForm from '../../components/atoms/ModalForm';

const defaultProps = {
    className: 'modal-class',
    style: { width: '100px' }
};

describe('Expect <ModalForm>', () => {
    it('to render', () => {
        const wrapper = mount(
            <ModalForm {...defaultProps}>
                <div>My form content</div>
            </ModalForm>
        );
        expect(wrapper).toMatchSnapshot();
    });
});
