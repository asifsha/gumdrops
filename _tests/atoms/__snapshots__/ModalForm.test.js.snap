// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect <ModalForm> to render 1`] = `
<ModalForm
  className="modal-class"
  style={
    Object {
      "width": "100px",
    }
  }
>
  <form
    className="gds-modal__form modal-class"
    style={
      Object {
        "width": "100px",
      }
    }
  >
    <div>
      My form content
    </div>
  </form>
</ModalForm>
`;
